src/tiledb/cloud/bioimg/ingestion.py

```diff
@@ -169,7 +169,7 @@ def iter_paths(sequence) -> Iterator[Tuple]:
 
     if len(source) == 1 and vfs.is_dir(source[0]):
         # Folder like input
-        return tuple(iter_paths(vfs.ls(source[0])))
+        return tuple(iter_paths(vfs.ls(source[0])[1:]))
```

Comment:
Do you need a length check here? What if the bucket is empty?

Reply:
That is correct, thanks! We were assuming already that there were some images to be ingested. But let's catch this edge case too.